deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -116,7 +117,13 @@
 is_enabled() ->
     rabbit_feature_flags:is_enabled(quorum_queue).
 
-%%----------------------------------------------------------------------------
+-spec is_compatible(boolean(), boolean(), boolean()) -> boolean().
+is_compatible(_Durable = true,
+              _Exclusive = false,
+              _AutoDelete = false) ->
+    true;
```

Comment:
Should it depend on the state of the `quorum_queue` feature flag too?

Reply:
Honestly, I would much rather emit an error at vhost creation time

Reply:
After some more thoughts, I now think that's the responsibility of the caller to call both `is_compatible()` **and** `is_enabled()`. The compatibility should not change with the state of the feature flag. Only the actual use of a queue type should be affected.